vscode-dotnet-runtime-library/src/Acquisition/DotnetConditionValidator.ts

```diff
@@ -143,7 +143,7 @@ Please set the PATH to a dotnet host that matches the architecture ${requirement
             return [];
         }
 
-        const findSDKsCommand = CommandExecutor.makeCommand(`"${existingPath}"`, ['--list-sdks', '--arch', requestedArchitecture]);

```

Comment:
Does this break Linux/Mac scenarios in the shell? I think we need to look at this from a broader context.